app/models/participant_outcome.rb

```diff
@@ -1,6 +1,12 @@
 class ParticipantOutcome < ApplicationRecord
   belongs_to :declaration
 
+  enum state: {
```

Comment:
note that this has already been added in another pr https://github.com/DFE-Digital/npq-registration/pull/1513/files#diff-bd21f06a16ff911a3e6127ef0154f4712f292420f86ce416314274a32bb0dee3R11

Reply:
Hi @cwrw Remove it from here then tksss 👍 😄